examples/cockroachdb/cockroachdb-blueprint.yaml

```diff
@@ -0,0 +1,126 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata:
+  name: cockroachdb-blueprint
+actions:
+  backup:
+    secretNames:
+      - cockroachSecret
+    outputArtifacts:
+      cockroachDBCloudDump:
+        keyValue:
+          s3path: '/cockroachdb-backups/{{ .StatefulSet.Namespace }}/{{ index .Object.metadata.labels "app.kubernetes.io/instance" }}/{{ toDate "2006-01-02T15:04:05.999999999Z07:00" .Time  | date "2006-01-02T15-04-05" }}'
+    phases:
+      - func: KubeTask
+        name: backupToS3
+        args:
+          namespace: "{{ .StatefulSet.Namespace }}"
+          image: "cockroachdb/cockroach"
+          podOverride:
+            containers:
+              - name: container
+                volumeMounts:
+                  - mountPath: /cockroach/cockroach-certs/
+                    name: client-certs
+            volumes:
+              - name: client-certs
+                projected:
+                  sources:
+                    - secret:
+                        name: '{{ .Secrets.cockroachSecret.Name }}'
+                        items:
+                          - key: ca.crt
+                            path: ca.crt
+                          - key: tls.crt
+                            path: client.root.crt
+                          - key: tls.key
+                            path: client.root.key
+                  defaultMode: 256
+
+          command:
+            - bash
+            - -o
+            - errexit
+            - -o
+            - xtrace
+            - -o
+            - pipefail
+            - -c
+            - |
+              host='{{ .StatefulSet.Name }}'-public
+              aws_access_key_id='{{ toJson .Profile.Credential.KeyPair.ID | toString }}'
+              aws_access_key_id=`echo $aws_access_key_id | sed 's/.\(.*\)/\1/' | sed 's/\(.*\)./\1/'`
```

Comment:
why do we need to trim this? Can we use the value from `aws_access_key_id` as is?

Reply:
@r4rajat can we use `tr -d \"` instead?

Reply:
Done